cves/CVE-2013-2115.yml

```diff
@@ -162,37 +184,43 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      if the OGNL successfully got through, the privileges of the user
```

Comment:
How would the privileges of the user be changed? 

I feel as though there is a misunderstanding of least privilege. In my mind, least privilege is the act of giving the apache or tomcat user minimal privileges needed to successfully run the application. This way, if the server is compromised, minimal damage can be done by the malicious actor.